cs_api_core/cs_api_core.ml

```diff
@@ -1,3 +1,5 @@
+module Log = Log
+
```

Comment:
**Question:* Is there any good reason to keep this?

This module doesn't log anything, so we could probably remove that. What's your opinion on this?